pennylane_cirq/simulator_device.py

```diff
@@ -109,11 +133,18 @@ def _apply_qubit_state_vector(self, qubit_state_vector_operation):
             )
 
         state_vector = np.array(qubit_state_vector_operation.parameters[0], dtype=np.complex64)
+        try:
+            wires = self.map_wires(qubit_state_vector_operation.wires)
+        except WireError:
+            wires = qubit_state_vector_operation.wires
```

Comment:
I'm not sure why this is, but there are cases when `QubitStateVector` is called on more wires than the device has. E.g., where it's called on wires `[0, 1]` but the device only has 1 wire.

```python
def test_var_single_wire_no_parameters(
    self, simulator_device_1_wire, tol, operation, input, expected_output
):
    """Tests that variances are properly calculated for single-wire observables without parameters."""

    op = operation(0, do_queue=False)

    simulator_device_1_wire.reset()
    simulator_device_1_wire.apply(
        [qml.QubitStateVector(np.array(input), wires=[0, 1])],
        rotations=op.diagonalizing_gates(),
    )
```

This causes issues with `map_wires` (thus the try-except) and also the `_expand_state` below (thus the `if len(wires) < self.num_wires ...` instead of `if len(wires) != self.num_wires ...`, as I first wrote).

Reply:
@thisac I would consider this a bug in the tests :) I suggest:

- removing the try-except here
- updating the test to use a 2 wire device